lxd/storage/backend_lxd.go

```diff
@@ -3135,10 +3135,20 @@ func (b *lxdBackend) EnsureImage(fingerprint string, op *operations.Operation) e
 		}
 	}
 
-	// Validate config and create database entry for new storage volume.
-	err = VolumeDBCreate(b, project.Default, fingerprint, "", drivers.VolumeTypeImage, false, volConfig, time.Now().UTC(), time.Time{}, contentType, false)
+	// For drivers which support both block mode and non-block mode, there should always only be a
```

Comment:
I'm not understanding this? Why not have separate records with the correct content type?